cmd/hare/oracle.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
add package desc for godoc

Reply:
This particular package is not part of this PR, I just moved stuff there.

I will add it for the `oracle` package, though.